core/src/main/resources/swagger/rawls.yaml

```diff
@@ -776,40 +776,6 @@ paths:
             - openid
             - email
             - profile
-    post:
```

Comment:
removed this api (with Alex's ok) because we will never really use it again and it is too hard to maintain